components/o-comments/src/scss/coral-talk-iframe/main.scss

```
@@ -21,6 +21,8 @@
 	}
 	#coral {
 		--ft-magnify: 1.31282;
+		display: inline-block!important;
```

Suggestion: let's fix this for now but report it to Coral, we can remove this code when they have fixed it their side

```suggestion
                 // See https://financialtimes.atlassian.net/jira/software/c/projects/CI/boards/1653?view=detail&selectedIssue=CI-1725
		display: inline-block !important;
```